engine/table/src/main/java/io/deephaven/engine/table/impl/SortOperation.java

```diff
@@ -26,14 +26,21 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-import java.util.Arrays;
-import java.util.LinkedHashMap;
-import java.util.Map;
+import java.util.*;
 import java.util.function.LongUnaryOperator;
 
 import static io.deephaven.engine.table.Table.SORT_REVERSE_LOOKUP_ATTRIBUTE;
+import static io.deephaven.engine.table.Table.SORT_ROW_REDIRECTION_ATTRIBUTE;
 
 public class SortOperation implements QueryTable.MemoizableOperation<QueryTable> {
+    static final Map<String, Object> IDENTITY_REDIRECTION_ATTRIBUTES;
+    // The "+" sign is not valid in a column, therefore we can be sure that this is a proper sentinel value.
+    static final String IDENTITY_REDIRECTION_VALUE = "+IDENTITY_REDIRECTION";
```

Comment:
Out of curiousity, do we do this `+` indication anywhere else? Seems useful but I haven't seen it.

Reply:
Not that I'm aware of, I just wanted a sentinel that worked and would not possibly conflict.  Since we require them in formulas, the + is definitely safe [since Java identifiers can't have it].